capture_node:
  ros__parameters:
    Kp_steering : 0.8 
    Ki_steering : 0.0 
    Kd_steering : 0.0 
    zero_throttle  : 0 
    max_throttle : 0.3
    min_throttle : 0.2
    error_threshold : 0.16 
    max_right_steering : 0.75
    max_left_steering : -0.75
robocar_align_node:
  ros__parameters:
    Kp_steering : 0.8 
    Ki_steering : 0.0 
    Kd_steering : 0.0 
    zero_throttle  : 0 
    max_throttle : 0.4
    min_throttle : 0.3
    error_threshold : 0.16 
    max_right_steering : 0.75
    max_left_steering : -0.75
intake_system_node:
  ros__parameters:
    bus_num : 1
    fan_channel : 13
collision_avoidance_node:
  ros__parameters:
    r_outer: .5
    r_inner: .15
    r_reverse: .25